caffe2/utils/proto_utils.cc

```diff
@@ -156,6 +166,14 @@ bool ParseFromString(const string& spec, Message* proto) {
 }
 } // namespace TextFormat
 
+bool ParseProtobufFromLargeString(const string& str, Message* proto) {
```

Comment:
Nit: Any chance you want to call this `ParseProtoFromLargeString` to match naming in here?

Reply:
Good point, will do.